llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -2927,6 +2927,24 @@ static X86::CondCode TranslateIntegerX86CC(ISD::CondCode SetCCOpcode) {
   }
 }
 
+static X86::CondCode getInvertedX86CC(X86::CondCode CC) {
```

Comment:
We already have this in X86InstroInfo as X86::GetOppositeBranchCondition - can we share a helper in X86BaseInfo.h?

Reply:
Great, thank! I had a quick search but didn't find it 🤦‍♀️